setup.py

```diff
@@ -65,6 +64,6 @@ def get_reqs(*fns):
         'Framework :: Django',
     ],
     zip_safe=False,
-    install_requires=get_reqs('requirements-min-django.txt', 'requirements.txt'),
```

Comment:
I believe the build failed because I forgot to remove this reference to the now-deleted file.